tests/Crypto/tst_crypto/tst_crypto.cpp

```diff
@@ -46,36 +78,50 @@ private slots:
+        QTest::newRow("AES CFB-128 128-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCfb128 << CryptoManager::EncryptionPaddingNone << 128;
+        QTest::newRow("AES CFB-128 192-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCfb128 << CryptoManager::EncryptionPaddingNone << 192;
+        QTest::newRow("AES CFB-128 256-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCfb128 << CryptoManager::EncryptionPaddingNone << 256;
 
-        QTest::newRow("CFB-8 128-bit") << CryptoManager::BlockModeCfb8 << 128;
-        QTest::newRow("CFB-8 192-bit") << CryptoManager::BlockModeCfb8 << 192;
-        QTest::newRow("CFB-8 256-bit") << CryptoManager::BlockModeCfb8 << 256;
+        QTest::newRow("AES OFB 128-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeOfb << CryptoManager::EncryptionPaddingNone << 128;
+        QTest::newRow("AES OFB 192-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeOfb << CryptoManager::EncryptionPaddingNone << 192;
+        QTest::newRow("AES OFB 256-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeOfb << CryptoManager::EncryptionPaddingNone << 256;
 
-        QTest::newRow("CFB-128 128-bit") << CryptoManager::BlockModeCfb128 << 128;
-        QTest::newRow("CFB-128 192-bit") << CryptoManager::BlockModeCfb128 << 192;
-        QTest::newRow("CFB-128 256-bit") << CryptoManager::BlockModeCfb128 << 256;
+        QTest::newRow("AES CTR 128-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCtr << CryptoManager::EncryptionPaddingNone << 128;
+        QTest::newRow("AES CTR 192-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCtr << CryptoManager::EncryptionPaddingNone << 192;
+        QTest::newRow("AES CTR 256-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCtr << CryptoManager::EncryptionPaddingNone << 256;
 
-        QTest::newRow("OFB 128-bit") << CryptoManager::BlockModeOfb << 128;
-        QTest::newRow("OFB 192-bit") << CryptoManager::BlockModeOfb << 192;
-        QTest::newRow("OFB 256-bit") << CryptoManager::BlockModeOfb << 256;
+        QTest::newRow("CTR 128-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCtr << CryptoManager::EncryptionPaddingNone << 128;
+        QTest::newRow("CTR 192-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCtr << CryptoManager::EncryptionPaddingNone << 192;
+        QTest::newRow("CTR 256-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCtr << CryptoManager::EncryptionPaddingNone << 256;
 
-        QTest::newRow("CTR 128-bit") << CryptoManager::BlockModeCtr << 128;
-        QTest::newRow("CTR 192-bit") << CryptoManager::BlockModeCtr << 192;
-        QTest::newRow("CTR 256-bit") << CryptoManager::BlockModeCtr << 256;
+        QTest::newRow("GCM 128-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeGcm << CryptoManager::EncryptionPaddingNone << 128;
```

Comment:
Should this one be called AES GCM 128-bit for consistency?  Same for the other key sizes and also for CCM?